templates/taobao-mini-game/patch.json

```diff
@@ -0,0 +1,3 @@
+{
+    "plugin-version": "3.7.3"
```

Comment:
If the version number is hardcoded, does that mean it needs to be updated for every version?
@PPpro @bofeng-song

Reply:
The editor only detects the existence of patch.json and does not use the internal version number